src/main/java/com/google/devtools/build/lib/runtime/UiOptions.java

```diff
@@ -276,6 +278,15 @@ public boolean useCursorControl() {
     return useCursesEnum == UseCurses.YES || (useCursesEnum == UseCurses.AUTO && isATty);
   }
 
+  public ImmutableSet<EventKind> getFilteredEvents() {
```

Comment:
rename method to make it more clear that this doesn't return events, e.g. to `getCombinedEventFilter` or `getFilteredEventKinds`